gc_registry/certificate/routes.py

```diff
@@ -26,8 +27,22 @@ def create_certificate_bundle(
     write_session: Session = Depends(db.get_write_session),
     read_session: Session = Depends(db.get_read_session),
     esdb_client: EventStoreDBClient = Depends(events.get_esdb_client),
+    nonce: str = "",
 ):
     """Create a GC Bundle with the specified properties."""
+
+    # Bundle issuance ID is the unique combination of device ID,
+    # energy carrier, and production starting interval.
+    certificate_bundle.issuance_id = f"""
```

Comment:
Should this not be a composite primary key rather than a string concatenation? Also not sure we need energy carrier. The implication of including it is that you can have different energy carriers for the same device in the same period is that a use case that we need?

Reply:
We've discussed this on the working group and with Killian a couple of times, making it a concatenated string aligns with the way most registries do it currently and it human-readable, which wouldn't be the case with a composite key. And yeah energy carrier is necessary because you can have e.g. co-located electrolysis plants that would claim GCs for both hydrogen and generation, but without device sub-metering. Also chp plants should heat ever get GCs assigned in the future